acceptance/tests/pxp-module-puppet/run_puppet_result_changed.rb

```diff
@@ -0,0 +1,65 @@
+require 'pxp-agent/test_helper.rb'
+
+ENVIRONMENT_NAME = 'changed_result'
+MODULE_NAME = 'changed'
+
+test_name 'C93062 - Run puppet and expect \'changed\' result' do
+
+  step 'On master, create a new environment that will result in Changed' do
+    environmentpath = master.puppet['environmentpath']
+    site_manifest = "#{environmentpath}/#{ENVIRONMENT_NAME}/manifests/site.pp"
+    on(master, "cp -r #{environmentpath}/production #{environmentpath}/#{ENVIRONMENT_NAME}")
+    create_remote_file(master, site_manifest, <<-SITEPP)
+node default {
+  include #{MODULE_NAME}
+}
+SITEPP
+    on(master, "chmod 644 #{site_manifest}")
+    on(master, "mkdir -p #{environmentpath}/#{ENVIRONMENT_NAME}/modules/#{MODULE_NAME}/manifests")
+    module_manifest = "#{environmentpath}/#{ENVIRONMENT_NAME}/modules/#{MODULE_NAME}/manifests/init.pp"
+    create_remote_file(master, module_manifest, <<-MODULEPP)
+class #{MODULE_NAME} {
+  notify {'Notify resources cause a Puppet run to have a \'changed\' outcome':}
+}
+MODULEPP
+    on(master, "chmod 644 #{module_manifest}")
+  end
+
+  step 'Ensure each agent host has pxp-agent running and associated' do
+    agents.each_with_index do |agent, i|
+      on agent, puppet('resource service pxp-agent ensure=stopped')
+      cert_dir = configure_std_certs_on_host(agent)
+      create_remote_file(agent, pxp_agent_config_file(agent), pxp_config_json_using_test_certs(master, agent, i + 1, cert_dir).to_s)
+      on agent, puppet('resource service pxp-agent ensure=running')
+      assert(is_associated?(master, "pcp://client0#{i+1}.example.com/agent"),
+             "Agent #{agent} with PCP identity pcp://client0#{i+1}.example.com/agent should be associated with pcp-broker")
+    end
+  end
+
+  step "Send an rpc_blocking_request to all agents" do
+    target_identities = []
+    agents.each_with_index do |agent, i|
+      target_identities << "pcp://client0#{i+1}.example.com/agent"
+    end
+    responses = nil # Declare here so not local to begin/rescue below
+    begin
+      responses = rpc_blocking_request(master, target_identities,
+                                      'pxp-module-puppet', 'run',
+                                      {:env => [], :flags => ['--server', "'#{master}'",
+                                                              '--onetime',
+                                                              '--no-daemonize',
+                                                              "--environment", "'#{ENVIRONMENT_NAME}'"]
```

Comment:
`"'#{ENVIRONMENT_NAME}'"` is a lot of overquoting.  Just ENVIRONMENT_NAME should work.